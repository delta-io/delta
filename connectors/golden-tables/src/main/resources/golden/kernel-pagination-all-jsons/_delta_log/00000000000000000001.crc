{"txnId":"f460664f-8dbf-4775-8a06-0ed95a058c6e","tableSizeBytes":5272,"numFiles":10,"numMetadata":1,"numProtocol":1,"setTransactions":[],"domainMetadata":[],"metadata":{"id":"b1624866-6060-42ab-9bbc-831735240dd3","format":{"provider":"parquet","options":{}},"schemaString":"{\"type\":\"struct\",\"fields\":[{\"name\":\"id\",\"type\":\"long\",\"nullable\":true,\"metadata\":{}}]}","partitionColumns":[],"configuration":{},"createdTime":1752013249002},"protocol":{"minReaderVersion":1,"minWriterVersion":2},"allFiles":[{"path":"part-00002-021db285-9b02-4973-a282-afcfd8afd007-c000.snappy.parquet","partitionValues":{},"size":527,"modificationTime":1752013250650,"dataChange":false,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":20},\"maxValues\":{\"id\":29},\"nullCount\":{\"id\":0}}"},{"path":"part-00004-39753157-19bf-4a6d-b65d-04505516762d-c000.snappy.parquet","partitionValues":{},"size":527,"modificationTime":1752013250722,"dataChange":false,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":40},\"maxValues\":{\"id\":49},\"nullCount\":{\"id\":0}}"},{"path":"part-00003-d6736222-7d56-4b90-ad5e-abea2a47353d-c000.snappy.parquet","partitionValues":{},"size":528,"modificationTime":1752013252206,"dataChange":false,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":80},\"maxValues\":{\"id\":89},\"nullCount\":{\"id\":0}}"},{"path":"part-00001-5236d99c-44d0-4eb3-b9a6-c84066e73742-c000.snappy.parquet","partitionValues":{},"size":527,"modificationTime":1752013252154,"dataChange":false,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":60},\"maxValues\":{\"id\":69},\"nullCount\":{\"id\":0}}"},{"path":"part-00000-520b63c8-7004-4fb3-9a7d-6b1ee913d1ac-c000.snappy.parquet","partitionValues":{},"size":527,"modificationTime":1752013252130,"dataChange":false,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":50},\"maxValues\":{\"id\":59},\"nullCount\":{\"id\":0}}"},{"path":"part-00004-b0a130bc-96cd-4b89-8d24-253f9fea21cb-c000.snappy.parquet","partitionValues":{},"size":527,"modificationTime":1752013252230,"dataChange":false,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":90},\"maxValues\":{\"id\":99},\"nullCount\":{\"id\":0}}"},{"path":"part-00001-f7ad6879-5e8e-4ac2-817f-2dc95477d7d7-c000.snappy.parquet","partitionValues":{},"size":527,"modificationTime":1752013250614,"dataChange":false,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":10},\"maxValues\":{\"id\":19},\"nullCount\":{\"id\":0}}"},{"path":"part-00002-113a09fa-8d92-4285-a192-75d9ef68ccdf-c000.snappy.parquet","partitionValues":{},"size":527,"modificationTime":1752013252182,"dataChange":false,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":70},\"maxValues\":{\"id\":79},\"nullCount\":{\"id\":0}}"},{"path":"part-00000-5aead402-bb3a-4c76-9d78-67c09cfcfa8a-c000.snappy.parquet","partitionValues":{},"size":528,"modificationTime":1752013250522,"dataChange":false,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":0},\"maxValues\":{\"id\":9},\"nullCount\":{\"id\":0}}"},{"path":"part-00003-a2307510-6cac-4772-8ad1-94e874534491-c000.snappy.parquet","partitionValues":{},"size":527,"modificationTime":1752013250686,"dataChange":false,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":30},\"maxValues\":{\"id\":39},\"nullCount\":{\"id\":0}}"}]}
