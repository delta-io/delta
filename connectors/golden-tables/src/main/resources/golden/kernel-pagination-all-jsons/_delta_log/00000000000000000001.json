{"commitInfo":{"timestamp":1752013252241,"operation":"WRITE","operationParameters":{"mode":"Append","partitionBy":"[]"},"readVersion":0,"isolationLevel":"Serializable","isBlindAppend":true,"operationMetrics":{"numFiles":"5","numOutputRows":"50","numOutputBytes":"2636"},"engineInfo":"Apache-Spark/3.5.3 Delta-Lake/3.4.0-SNAPSHOT","txnId":"f460664f-8dbf-4775-8a06-0ed95a058c6e"}}
{"add":{"path":"part-00000-520b63c8-7004-4fb3-9a7d-6b1ee913d1ac-c000.snappy.parquet","partitionValues":{},"size":527,"modificationTime":1752013252130,"dataChange":true,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":50},\"maxValues\":{\"id\":59},\"nullCount\":{\"id\":0}}"}}
{"add":{"path":"part-00001-5236d99c-44d0-4eb3-b9a6-c84066e73742-c000.snappy.parquet","partitionValues":{},"size":527,"modificationTime":1752013252154,"dataChange":true,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":60},\"maxValues\":{\"id\":69},\"nullCount\":{\"id\":0}}"}}
{"add":{"path":"part-00002-113a09fa-8d92-4285-a192-75d9ef68ccdf-c000.snappy.parquet","partitionValues":{},"size":527,"modificationTime":1752013252182,"dataChange":true,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":70},\"maxValues\":{\"id\":79},\"nullCount\":{\"id\":0}}"}}
{"add":{"path":"part-00003-d6736222-7d56-4b90-ad5e-abea2a47353d-c000.snappy.parquet","partitionValues":{},"size":528,"modificationTime":1752013252206,"dataChange":true,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":80},\"maxValues\":{\"id\":89},\"nullCount\":{\"id\":0}}"}}
{"add":{"path":"part-00004-b0a130bc-96cd-4b89-8d24-253f9fea21cb-c000.snappy.parquet","partitionValues":{},"size":527,"modificationTime":1752013252230,"dataChange":true,"stats":"{\"numRecords\":10,\"minValues\":{\"id\":90},\"maxValues\":{\"id\":99},\"nullCount\":{\"id\":0}}"}}
